### ISSUE TITLE
json-ld injected through <svelte:head> will be duplicated

### ISSUE THREAD

**Issue by winston0410** [sveltejs/kit#4587]
### Describe the bug

simple json object of `json-ld` will be duplicated with injecting into the document with `<svelte:head>`. The tag will be rendered twice.

I use a function like this:

```typescript
export const mkSchema = (thing: Schema) => {
	return `<script type="application/ld+json">${JSON.stringify(thing, null, 2)}</script>`;
};
```

And render like this:

```svelte
<svelte:head>
    {@html mkSchema(orgSchema)}
</svelte:head>
```

### Reproduction

https://github.com/winston0410/duplicate-head

### System Info

```shell
System:
    OS: macOS 11.4
    CPU: (8) arm64 Apple M1
    Memory: 123.59 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 17.8.0 - /opt/homebrew/bin/node
    Yarn: 1.22.17 - /opt/homebrew/bin/yarn
    npm: 8.5.5 - /opt/homebrew/bin/npm
  Browsers:
    Brave Browser: 99.1.36.119
    Safari: 14.1.1
  npmPackages:
    @sveltejs/adapter-auto: next => 1.0.0-next.34 
    @sveltejs/kit: next => 1.0.0-next.310 
    svelte: ^3.44.0 => 3.47.0
```

### Severity

blocking all usage of SvelteKit

### Additional Information

It will greatly affect the SEO of sveltekit as structured data cannot be used

**Comment by dominikg** [comment]
Does it also happen  when you put the script tag into `<svelte:head>` directly without using `@html` ?

note: to avoid potential security issues/rendering errors, you should escape the content of the script tag ( see eg. https://github.com/sveltejs/kit/blob/01221d30a78483a212ccb2d3e9b2ee0094b916cb/packages/kit/src/utils/escape.js#L44)

**Comment by winston0410** [comment]
@dominikg It will not happen if you do that directly, but then the content of the object cannot be expanded.

A repro for that behavior:

https://github.com/winston0410/duplicate-head/tree/use-head

**Comment by dominikg** [comment]
looks like you already found https://github.com/sveltejs/svelte/issues/1607 and posted there before opening this issue.

This information should have been included in your original report.

**Comment by winston0410** [comment]
Yup I have found that. Should I proceed this issue here, or do it over that one? Because that one is closed right now.
